src/basho_bench.erl

```diff
@@ -96,8 +96,8 @@ run_benchmark(Configs) ->
     basho_bench_sup:start_child(),
     ok = basho_bench_stats:run(),
     ok = basho_bench_measurement:run(),
-    ok = basho_bench_worker:run(basho_bench_worker_sup:workers()),
     ok = basho_bench_duration:run(),
+    ok = basho_bench_worker:run(basho_bench_worker_sup:workers()),
     application:set_env(basho_bench_app, is_running, true).
 
 
```

Comment:
update invoke sequence of worker run and duration run gen_server and this will make the ops counter is more accurate or else will missing about 10 second data.